paper.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'scikit-COSMO: A Toolbox of Machine Learning Methods for Materials Science'
+tags:
+    - Python
+authors:
+    - name: Rose K. Cersonsky ^ [\url{mailto: rose.cersonsky@epfl.ch}]
+    orcid: 0000-0003-4515-3441
+    affiliation: 1
+    - name: Guillaume Fraux
+    orcid: 0000-0003-4824-6512
+    affiliation: 1
+    - name: Sergei Kliavinek
+    orcid: 0000-0001-8326-325X
+    affiliation: 1
+    - name: Alexander Goscinski
+    orcid: 0000-0001-8076-215X
+    affiliation: 1
+    - name: Benjamin A. Helfrecht
+    orcid: 0000-0002-2260-7183
+    affiliation: 1
+    - name: Michele Ceriotti ^ [\url{mailto: michele.ceriotti@epfl.ch}]
+    orcid: 0000-0003-2571-2832
+    affiliation: 1
+affiliations:
+    - name: Laboratory of Computational Science and Modeling(COSMO), École Polytechnique Fédérale de Lausanne(EPFL), Lausanne, Switzerland
+    index: 1
+date: 03 May 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`scikit-COSMO` (skcosmo) focuses on machine learning methods that are of particular use (but not limited
+                                                                                        to) the field of materials science and computational chemistry. Currently, many
+machine learning studies employ the foundational `scikit-learn` repository, a
+collection of widely applicable machine learning algorithms and methods.
+Written in the same style and guidelines as `scikit-learn`, we aim to provide
+users the ability to seamlessly include these methods in machine learning
+workflows alongside those from `scikit-learn`.
+
+# Statement of need
+
+In `scikit-COSMO`, we provide machine learning algorithms developed for a specific
+community: those who use machine learning to understand or represent collections
+of atoms or molecules. The emphasis put upon each subclass of machine learning
+algorithms varies within different subfields. For example, in applying machine
+learning to collections of atoms, there is a necessary focus on the
+representation of data in a numerical format for machine learning pipelines,
+known as _featurization_. Many machine learning studies represent samples using
+real-world measurements or tests, effectively limiting the number of features
+based upon the resources available. When representing a collection of atoms,
+there is both a variety of featurization techniques to employ and a wealth of
```

Comment:
We could add a small list of such representation & parameters, and link to the tree of representation paper here.